### ISSUE TITLE
APIView should keep revisions for release packages

### ISSUE THREAD

**Issue by pakrym** [Azure/azure-sdk-tools#1770]
Take for example:

https://apiview.dev/Assemblies/Review/b67e68cfee4841bda045f895c90ad383?doc=False

We have 14 preview versions at this point but only 2 revisions in API view. Makes it very hard to follow diffs.

cc @weshaggard @praveenkuttappan

**Comment by praveenkuttappan** [comment]
Currently we retain only approved revisions in automatic reviews. But we can change this by tagging a revision to release version

**Comment by WhitShake** [comment]
We currently tag a revision as released as part of release and retain
